tests/ttnn/unit_tests/operations/test_all_gather_N300_post_commit.py

```diff
@@ -0,0 +1,386 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import pytest
+from loguru import logger
+import ttnn
+from tests.tt_eager.python_api_testing.sweep_tests.comparison_funcs import comp_equal, comp_pcc
+from models.utility_functions import skip_for_grayskull
+
+
+def is_unsupported_case(input_shape, dim, mem_config, num_devices, num_links, input_dtype, layout):
```

Comment:
This looks like it's still the same check as in test_all_gather.py. Please import and use that one instead.